src/contrib/dependencyInjection/Akka.DI.TestKit/DiResolverSpec.cs

```diff
@@ -335,7 +336,7 @@ public void DependencyResolver_should_inject_into_normal_mailbox_Actor()
 
             var internalRef = (LocalActorRef)stashActor;
 
-            Assert.IsType<UnboundedMailbox>(internalRef.Cell.Mailbox);
+            Assert.IsType<UnboundedMessageQueue>(internalRef.Cell.Mailbox.MessageQueue);
```

Comment:
There is one and only one `Mailbox` implementation, but it can use different message queues. Test has been adjusted to reflect that.